### PR TITLE
added Dockerfile, docker-compose.yml, .env file

### PR DESCRIPTION
**Pull request by IhorKalaur**

### CODE REVIEW

**Review by irynamekh — APPROVED** [review]
Good job)

File: .env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=123
+MYSQLDB_DATABASE=test
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (nacenik) [inline .env]: don't forget about empty line

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url"  : "jdbc:mysql://mysqldb:$MYSQLDB_DOCKER_PORT/$MYSQLDB_DATABASE?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useSSL=false&useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC",
+        "spring.datasource.username" : "$MYSQLDB_USER",
+        "spring.datasource.password" : "$MYSQLDB_ROOT_PASSWORD",
+        "spring.jpa.properties.hibernate.dialect" : "org.hibernate.dialect.MySQL8Dialect",
+        "spring.jpa.hibernate.ddl-auto" : "validate"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (nacenik) [inline docker-compose.yml]: empty line

**Review by nacenik — CHANGES_REQUESTED** [review]
GJ, Let's add some improvements

**Review by nacenik — APPROVED** [review]
GJ! :)

**Pull request merged**